### ISSUE TITLE
[no-standalone-expect] False positive inside it.fails

### ISSUE THREAD

**Issue by KoenBrouwer** [veritem/eslint-plugin-vitest#275]
**Summary**
`vitest/no-standalone-expect` seems to wrongly warn about an expect that is not standalone.

**Steps to reproduce**
Use Vitest with `vitest/no-standalone-expect` enabled in ESLint, run the following test:

```json
// .eslintrc

{
    "plugins": [
        "vitest",
    ],
    "rules": {
        "vitest/no-standalone-expect": [
            "error"
        ],
    },
}

```

```javascript
// test.ts

it.fails("should work", () => {
    expect(true).toBe(false);
});
```

**What you expected would happen**
ESLint shouldn't error on the expect, as it clearly is inside a test/it block.

**What actually happens**
ESLint warns/errors (depending on your setting):

```javascript
it.fails("should work", () => {
    expect(true).toBe(false);
     ^^^ ESLint: Expect must be called inside a test block(vitest/no-standalone-expect)
});
```

Maybe related to #182?

**Comment by veritem** [comment]
a pr is welcome!

**Comment by KoenBrouwer** [comment]
> a pr is welcome!

Ah well, alright. Here you go: https://github.com/veritem/eslint-plugin-vitest/pull/276
